packages/editor/src/components/inserter/index.js

```diff
@@ -88,26 +88,31 @@ class Inserter extends Component {
 }
 
 export default compose( [
-	withSelect( ( select ) => {
+	withSelect( ( select, { rootClientId, layout } ) => {
 		const {
 			getEditedPostAttribute,
 			getBlockInsertionPoint,
 			getSelectedBlock,
 			getInserterItems,
+			getBlockOrder,
 		} = select( 'core/editor' );
 		const insertionPoint = getBlockInsertionPoint();
-		const { rootClientId } = insertionPoint;
+		const parentId = rootClientId || insertionPoint.rootClientId;
 		return {
 			title: getEditedPostAttribute( 'title' ),
-			insertionPoint,
+			insertionPoint: {
```

Comment:
We should almost never want to create a new object to be returned as a value of `mapSelectToProps` return value object.

See: #11028